hawk/app/assets/stylesheets/authentication/_content.scss

```diff
@@ -32,13 +31,17 @@
     margin: 50px 20px 55px 20px;
 
     a {
-      color: #00b268;
+      color: #059aaa;
       text-decoration: none;
 
       &:hover {
-        color: #00b268;
+        color: #059aaa;
```

Comment:
Color literals like `#059aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.